cometsdk.go

```diff
@@ -4087,23 +4119,29 @@ func (this *CometAPIClient) AdminAccountSessionStart(SelfAddress *string) (*Sess
 	if err != nil {
 		return nil, err
 	}
+	if c.ReuseSessionKey {
```

Comment:
Every SessionStart API check the ReuseSessionKey option before automatically setting the SessionKey on the client.

Reply:
I like this design, I think it's very easy to use and it solves the backward compatibility concern too.